src/components/forms/Select/Select.stories.tsx

```diff
@@ -33,6 +33,7 @@ const options = (
     <option value="value1">Option A</option>
     <option value="value2">Option B</option>
     <option value="value3">Option C</option>
+    <option value="valueBIG">Option of extra length to demonstrate how content like this will look different</option>
```

Comment:
Looks like the build is failing due to prettier:

<img width="744" alt="image" src="https://github.com/trussworks/react-uswds/assets/15805554/25a7e980-a1da-45b9-bc2b-60b49ca65252">

I imagine prettier will want this line shorter, it'll probably add some newlines or something, but give it a run locally and then push up the changes, and this should be good to go

Reply:
oh weird, i ran it and it said it was fine locally

Reply:
ah no i didn't, i ran lint

Reply:
ok fixed